### DISCUSSION TITLE
Excalidraw

### DISCUSSION THREAD

**Discussion by johannSo** [catppuccin/catppuccin#2626] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

I think Excalidraw can be easily themed with Stylus CSS

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by isitreallyalive** [comment]
Did you ever get around to working on this? I would love to see it happen!

**Comment by nuexq** [comment]
It's not possible to theme Excalidraw because of the canvas. The colors inside the canvas, like shapes, text, and drawings, are just js variables, and we can't change/access them with CSS.

**Comment by WalkQuackBack** [comment]
Made a PR for this: https://github.com/catppuccin/userstyles/pull/1961
